src/languages/en.ts

```diff
@@ -1046,7 +1046,7 @@ const translations = {
         canceled: 'Canceled',
         posted: 'Posted',
         deleteReceipt: 'Delete receipt',
-        deletedTransaction: ({amount, merchant}: DeleteTransactionParams) => `deleted an expense on this report (${merchant} - ${amount})`,
+        deletedTransaction: ({amount, merchant}: DeleteTransactionParams) => `deleted an expense (${amount} for ${merchant})`,
```

Comment:
Wait, @jamesdeanexpensify had these  reversed at his pr 😅 what is the correct one?

Reply:
amount for merchant, sorry!

Reply:
Thanks for confirming!